packages/contracts/hardhat.config.ts

```diff
@@ -14,6 +14,9 @@ const config: HardhatUserConfig = {
     version: '0.8.15',
   },
   networks: {
+    hardhat: {
+      allowUnlimitedContractSize: true,
+    },
```

Comment:
I had to set this because the `ChugSplashBootLoader` contract is huge... gonna need to get it to compile somehow.